scripts/install.sh

```
@@ -112,33 +112,51 @@ install_dependencies() {
-    info "Downloading weed-volume ${SEAWEED_VERSION} (${RUST_ASSET})"
-    curl {{.ProxyConfig}} -o "$TMP_DIR/${RUST_ASSET}" -sfL "${RUST_URL}"
-    curl {{.ProxyConfig}} -o "$TMP_DIR/${RUST_ASSET}.md5" -sfL "${RUST_URL}.md5"
-    info "Verifying weed-volume ${SEAWEED_VERSION}"
-    md5Value=`cat "$TMP_DIR/${RUST_ASSET}.md5" | awk '{print $1}'`
-    ( cd "$TMP_DIR" && echo "${md5Value}  ${RUST_ASSET}" | md5sum -c )
+    info "Downloading weed-volume ${RUST_VERSION_ID} (${RUST_URL})"
+    curl {{.ProxyConfig}} --retry 3 --retry-delay 2 -o "$TMP_DIR/${RUST_ASSET}" -sfL "${RUST_URL}"
+    # Verify md5 when published. Stable releases ship a .md5; the rolling dev
+    # build currently does not, so a missing .md5 is best-effort rather than a
+    # hard failure (the download already came over HTTPS from the release).
+    if curl {{.ProxyConfig}} --retry 3 --retry-delay 2 -o "$TMP_DIR/${RUST_ASSET}.md5" -sfL "${RUST_MD5_URL}"; then
```

Shellcheck (0.11.0)

[warning] This { is literal. Check expression (missing ;/\n?) or quote it.

(SC1083)